src/components/_timeline.scss

```diff
@@ -0,0 +1,162 @@
+/// See https://www.nice.org.uk/about/what-we-do/our-programmes/nice-guidance/nice-diagnostics-guidance/process-timeline
+/// See https://www.nice.org.uk/about/what-we-do/our-programmes/nice-guidance/nice-technology-appraisal-guidance/charging/timelines-ta
+
+// Defines styles for < md viewports, and for when a single column timeline is required (e.g. here - https://www.nice.org.uk/about/what-we-do/life-sciences/scientific-advice)
+@mixin flat-style-timeline {
+  &:before {
+    left: 40px;
+  }
+  .timeline-panel {
+    width: calc(100% - 130px);
+    float: right;
+    &:before {
+      border-left-width: 0;
+      border-right-width: 15px;
+      left: -15px;
+      right: auto;
+    }
+    &:after {
+      border-left-width: 0;
+      border-right-width: 14px;
+      left: -14px;
+      right: auto;
+    }
+  }
+  .timeline-badge {
+    left: 16px;
+    margin-left: 0;
+    top: 16px;
+  }
+}
+
+.timeline {
+  @include mq($until: md) {
+    @include flat-style-timeline;
+  }
+
+  &--flat {
+    @include flat-style-timeline;
+  }
+
+  list-style: none;
+  position: relative;
+
+  // this is the line that goes down the centre of the timeline badges
+  &:before {
+    background-color: $nds-colour-nice-grey-2;
+    bottom: 0;
+    content: '';
+    left: 50%;
+    margin-left: -(rem($nds-spacing-xx-small));
+    position: absolute;
+    top: 0;
+    width: rem($nds-spacing-x-small);
+  }
+
+  > li {
+    @include nds-clearfix;
+    margin-bottom: rem($nds-spacing-medium);
+    position: relative;
+  }
+}
+
+.timeline-inverted {
+  .timeline-panel {
+    float: right;
+    &:before {
+      border-left-width: 0;
+      border-right-width: 15px;
+      left: -15px;
+      right: auto;
+    }
+    &:after {
+      border-left-width: 0;
+      border-right-width: 14px;
+      left: -14px;
+      right: auto;
+    }
+  }
+}
+
+.timeline-panel {
+  box-shadow: none !important;
+  background: $nds-colour-nice-white;
+  width: 42%;
+  float: left;
+  border: 1px solid $nds-colour-nice-grey-2;
+  border-radius: 2px;
+  padding: rem(20px);
+  position: relative;
+  &:before {
+    position: absolute;
+    top: 26px;
+    right: -15px;
+    display: inline-block;
+    border-top: 15px solid transparent;
+    border-left: 15px solid $nds-colour-nice-grey-2;
+    border-right: 0 solid $nds-colour-nice-grey-2;
+    border-bottom: 15px solid transparent;
+    content: '';
+  }
+  &:after {
+    position: absolute;
+    top: 27px;
+    right: -14px;
+    display: inline-block;
+    border-top: 14px solid transparent;
+    border-left: 14px solid $nds-colour-nice-white;
+    border-right: 0 solid $nds-colour-nice-white;
+    border-bottom: 14px solid transparent;
+    content: '';
+  }
+}
+
+.timeline-title {
+  margin-top: 0;
+}
+
+.timeline-badge {
+  color: $nds-colour-nice-white;
+  width: 50px;
+  height: 50px;
+  line-height: 50px;
+  text-align: center;
+  position: absolute;
+  top: 16px;
+  left: 50%;
+  margin-left: -25px;
```

Comment:
I wonder if here we can use our spacing values for the width and height, and using a negative x transform of 50% rather than a negative margin left?